### ISSUE TITLE
changing switch port settings fec does not actually changing link fec

### ISSUE THREAD

**Issue by rcgoodfellow** [oxidecomputer/omicron#5667]
I changed link fec through the API and it did not take place in dendrite. Only when I deleted the link and let the RPW recreate it did the fec mode get changed

**Comment by jgallagher** [comment]
Is this related to https://github.com/oxidecomputer/dendrite/issues/588? That one was about autonegotiation, but if the RPW is using `apply_port_settings` it might have the same root cause.

**Comment by bnaecker** [comment]
@rcgoodfellow Any chance this was resolved with your latest updates on the Dendrite side?

**Comment by rcgoodfellow** [comment]
I don't think so. We would have seen similar symptoms for equalization settings that oxidecomputer/dendrite#136 fixed, but I don't think my recent changes would impact fec. It is possible that a similar issue exists for fec, but i'd need to go test.
